R/dcflights21.R

```diff
@@ -13,14 +13,48 @@
 #'   \item{origin, destination}{Origin and destination airports.}
 #'   \item{date}{Date of departure.} 
 #'   \item{hour}{Time of scheduled departure by hour}
-#'   \item{dep_delay}{Departure delay, in
-#'     minutes. Negative values represent early departures/arrivals.}
 #'   \item{plane}{Plane tail number, giving a unique ID for each plane.} 
-#'   \item{air_time}{Scheduled amount of time spent in the air, in minutes.} 
-#'   \item{duration}{Distance between airports, in miles.} 
+#'   \item{distance}{Distance between airports, in miles.} 
+#'   \item{duration}{Scheduled amount of time spent in the air, in minutes.} 
 #'   \item{wind_speed}{Wind speed, in miles per hour.}
 #'   \item{visibility}{Visibility, in miles.}
-#'   \item{year}{Year the flown plane was manufactured.}
+#'   \item{plane_year}{Year the flown plane was manufactured.}
+#'   \item{dep_delay}{Departure delay, in
+#'     minutes. Negative values represent early departures/arrivals.}
+#' }
+#' @source RITA, Bureau of transportation statistics,
+#' \url{https://www.transtats.bts.gov/DL_SelectFields.asp?Table_ID=236}
+#' @source \url{https://www.transtats.bts.gov/DL_SelectFields.asp?Table_ID=236}
+#' @source ASOS download from Iowa Environmental Mesonet,
+#' \url{https://mesonet.agron.iastate.edu/request/download.phtml}.
+#' @source \url{https://www.transtats.bts.gov/DL_SelectFields.asp?Table_ID=236}
+#' @source FAA Aircraft registry,
+#' \url{http://www.faa.gov/licenses_certificates/aircraft_certification/aircraft_registry/releasable_aircraft_download/}
+#' @keywords datasets
+NULL
+
+
+#' Flights data
+#'
+#' Data for flights that departed from the Washington DC airport (Reagan 
+#' Internation) during July 2021 and arrived at one of the top 10 connecting
+#' airports.
+#'
+#' @name dcflights
+#' @aliases flights
```

Comment:
Seeing a warning on Actions about duplicated aliases—could you drop this line or the analogous one for the larger dataset and redocument? Thumbs up from me if you'd like them to share an `@rdname`, though.🙂

Reply:
I think I'd rather keep them in separate man files so that the first day, folks can just focus on this dataset only.